control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -565,10 +565,17 @@ func (r *reconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Result
 		errs = append(errs, fmt.Errorf("failed to reconcile the kube apiserver service monitor: %w", err))
 	}
 
+	disableMultiNetwork := false
```

Comment:
This logic is used in 2 other places. Maybe it's worth putting it in a separate function in the /support/util package that gets called from both hostedcontrolplane_controller and here.

Something like: `util.IsMultiNetworkDisabled(hcp *hyperv1.HostedControlPlane)`